Resources/Prototypes/Stacks/Materials/crystals.yml

```diff
@@ -3,3 +3,7 @@
   name: telecrystal
   icon: Objects/Specific/Syndicate/telecrystal.rsi
   spawn: Telecrystal1
+  components:
+  - type: Tag
+    tags:
+      - Telecrystal
```

Comment:
linter hates you for this. I don't know if it's actually possible to put tags on stacks?

Reply:
I'm pretty sure the stack inherits the tags of its component part but you would probably have to test that to be sure

Reply:
this does indeed make the bounty work, yes. I have tested